src/sagemaker/xgboost/defaults.py

```diff
@@ -17,3 +17,4 @@
 XGBOOST_VERSION_1 = "0.90-1"
 XGBOOST_LATEST_VERSION = "0.90-2"
 XGBOOST_SUPPORTED_VERSIONS = [XGBOOST_VERSION_1, XGBOOST_LATEST_VERSION]
+XGBOOST_LATEST_VERSION_EQUIVALENTS = [XGBOOST_LATEST_VERSION, XGBOOST_LATEST_VERSION + "-cpu-py3"]
```

Comment:
I like this idea. Since we need to check against `(version, version + "-cpu-py3")` for all versions, it may be better for us to have a helper function that returns the equivalents instead of a constant specifically for the latest version.

Reply:
Added a helper to generate the equivalent versions.